superset/translations/pt_BR/LC_MESSAGES/messages.json

```diff
@@ -1,2633 +1 @@
-         "Check to include Druid Granularity dropdown": [
-            "Verifique para incluir o menu suspenso Druid Granularity"
-         ],
-         "Show Druid Time Origin": [
-            "Show Druid Time Origin"
-         ],
-         "Check to include Time Origin dropdown": [
-            "Verifique para incluir o menu suspenso Origem do tempo"
-         ],
-         "Data Table": [
-            "Tabela de dados"
-         ],
-         "Whether to display the interactive data table": [
-            "Se deseja exibir a tabela de dados interativos"
-         ],
-         "Search Box": [
-            "Caixa de pesquisa"
-         ],
-         "Whether to include a client side search box": [
-            "Se incluir uma caixa de pesquisa do lado do cliente"
-         ],
-         "Table Filter": [
-            "Filtro de mesa"
-         ],
-         "Whether to apply filter when table cell is clicked": [
-            "Se deseja aplicar o filtro quando a célula da tabela é clicada"
-         ],
-         "Show Bubbles": [
-            "Show Bubbles"
-         ],
-         "Whether to display bubbles on top of countries": [
-            "Seja para exibir bolhas em cima dos países"
-         ],
-         "Legend": [
-            "lenda"
-         ],
-         "Whether to display the legend (toggles)": [
-            "Se deseja exibir a legenda (alternar)"
-         ],
-         "Show Values": [
-            "Mostrar valores"
-         ],
-         "Whether to display the numerical values within the cells": [
-            "Se deseja exibir os valores numéricos dentro das células"
-         ],
-         "X bounds": [
-            "X limites"
-         ],
-         "Whether to display the min and max values of the X axis": [
-            "Se deseja exibir os valores mínimo e máximo do eixo X"
-         ],
-         "Y bounds": [
-            "Y limites"
-         ],
-         "Whether to display the min and max values of the Y axis": [
-            "Se deseja exibir os valores mínimo e máximo do eixo Y"
-         ],
-         "Rich Tooltip": [
-            "Rich Tooltip"
-         ],
-         "The rich tooltip shows a list of all series for that point in time": [
-            "A ferramenta de informação rica mostra uma lista de todas as séries para esse ponto no tempo"
-         ],
-         "Y Log Scale": [
-            "Y Escala de registro"
-         ],
-         "Use a log scale for the Y axis": [
-            "Use uma escala de log para o eixo Y"
-         ],
-         "X Log Scale": [
-            "X Log Scale"
-         ],
-         "Use a log scale for the X axis": [
-            "Use uma escala de log para o eixo X"
-         ],
-         "Log Scale": [
-            "Escala logarítmica"
-         ],
-         "Use a log scale": [
-            "Use uma escala de registro"
-         ],
-         "Donut": [
-            "Rosquinha"
-         ],
-         "Do you want a donut or a pie?": [
-            "Você quer uma rosquinha ou uma torta?"
```

Comment:
Fica meio estranho assim

Reply:
hahaha ficou com duplo sentido... Mas ia ser  "Você quer uma rosca ou uma torta?" que também não seria bom... Talvez  "Você quer um gráfico de rosca ou de torta?", que tal?

Reply:
Better, let's continue this in English, yet the original text really says do you want a donut or a pie ;)